CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.9.0] - 2015-12-21
+### Added
+- Add package.json for npm support
```

Comment:
@jeremiak: @alexose mentioned liking having pre-built files in the repo so people can clone and get started right away. He also mentioned D3 as a good example that does this. cc @msecret